test/unit/ConfigurationTest.php

```diff
@@ -306,4 +313,60 @@ public function getValidatorShouldReturnTheConfiguredValidator(): void
 
         self::assertSame($this->validator, $config->getValidator());
     }
+
+    /**
+     * @test
+     *
+     * @covers \Lcobucci\JWT\Configuration::getValidationConstraints
+     *
+     * @uses \Lcobucci\JWT\Configuration::forUnsecuredSigner
+     * @uses \Lcobucci\JWT\Configuration::__construct
+     * @uses \Lcobucci\JWT\Signer\None
+     * @uses \Lcobucci\JWT\Signer\Key
+     */
+    public function getValidationConstraintsShouldReturnTheDefaultWhenItWasNotConfigured(): void
```

Comment:
I'd suggest to rename this method to `getValidationConstraintsShouldReturnAnEmptyArrayWhenItWasNotConfigured`, since it makes our test intention a bit more explicit